File: app/src/main/java/com/zivapp/notes/util/dialog/DialogDeleteNote.java
package com.zivapp.notes.util.dialog;

import android.app.Activity;
import android.app.AlertDialog;
import android.view.LayoutInflater;
import android.view.View;

import androidx.appcompat.app.AppCompatDialogFragment;

import com.zivapp.notes.R;

public class DialogDeleteNote extends AppCompatDialogFragment {

    private static final String TAG = "DialogDeleteNote";

    public static AlertDialog.Builder onCreateDialogBuilder(Activity activity) {
        AlertDialog.Builder builder = new AlertDialog.Builder(activity);

        LayoutInflater inflater = activity.getLayoutInflater();
        View view = inflater.inflate(R.layout.dialog_edit_note, null);

        builder.setView(view);

        return builder;
    }
}

File: Kotlin/app/src/main/java/com/zivapp/countandnote/util/dialog/DialogDeleteNote.kt
package com.zivapp.countandnote.util.dialog

import android.app.Activity
import android.app.AlertDialog
import android.view.View
import androidx.appcompat.app.AppCompatDialogFragment
import com.zivapp.countandnote.R

class DialogDeleteNote : AppCompatDialogFragment() {
    companion object {
        private const val TAG = "DialogDeleteNote"

        fun onCreateDialogBuilder(activity: Activity): AlertDialog.Builder? {
            val builder = AlertDialog.Builder(activity)
            val inflater = activity.layoutInflater
            val view: View = inflater.inflate(R.layout.dialog_edit_note, null)
            builder.setView(view)
            return builder
        }
    }
}